samples/cpp/tutorial_code/photo/seamless_cloning/cloning_gui.cpp

```diff
@@ -540,7 +540,7 @@ int main()
             imwrite("cloned.png",blend);
         }
         else if(key == 'q')
-            exit(0);
+            break;
     }
     waitKey(0);
```

Comment:
This line looks useless anyway. After pressing `q` you need to press some key to exit finally.